packages/camera/camera_windows/pubspec.yaml

```diff
@@ -0,0 +1,30 @@
+name: camera_windows
+description: A Flutter plugin for getting information about and controlling the camera on Windows.
+version: 0.0.1
+# TODO: Update next lines to match proper repository and issue_tracker url
+#repository: https://github.com/flutter/plugins/tree/master/packages/camera/camera_windows
+#issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
+# TODO: Remove next line after plugin is ready to be published
```

Comment:
What changes are planned before publishing?

Reply:
Ok, I added this here only to avoid possible accidental publishing on situations unknown for me...
Removed.